web/packages/teleport/src/features.tsx

```diff
@@ -208,6 +210,36 @@ export class FeatureUsers implements TeleportFeature {
   }
 }
 
+export class FeatureBots implements TeleportFeature {
+  category = NavigationCategory.Management;
+  section = ManagementSection.Access;
+
+  route = {
+    title: 'Manage Bots',
+    path: cfg.routes.bots,
+    exact: true,
+    component: () => <Bots />,
```

Comment:
I'm not sure where the `component: () =>` pattern came from but can this just be 
```suggestion
    component: Bots,
```
?